### PR TITLE
Added README.md

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+    sudo add-apt-repository -y ppa:fgimenez/snappy-cloud-image
+    sudo apt-get update && sudo apt-get install snappy-cloud-image
+
+# Requirements
+
+At the moment the binary only works for creating images to be used in an OpenStack environment. You should have the common OpenStack environment variables (`$OS_USERNAME`, `$OS_TENANT_NAME`, `$OS_PASSWORD`, `$OS_AUTH_URL` and `$OS_REGION_NAME`) loaded before executing the binary.
+
+# Getting help
+
+You take a look at the options of the command with:
```
Comment (come-maiz) [inline README.md]: You CAN take

File: README.md [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+
+Each time you invoke the `snappy-cloud-image` command you should pass an `-action` to it, which can be one of:
+
+## create
+
+This action does several things:
+
+* Determines if there's a new image to be created. For this it checks the source endpoint (at http://system-image.ubuntu.com) and the latest image version at the glance endpoint for a given combination of `-release`, `-channel` and `-arch`.
+
+* If there's a new version available then:
```
Comment (come-maiz) [inline README.md]: then IT WILL:

File: README.md [hunk 3]
```diff
@@ -1,4 +1,51 @@
...
+
+  * Create a new raw local image using ubuntu-device-flash
+
+  * Convert the raw image to QCOW2 format
+
+  * Upload to glance
```
Comment (come-maiz) [inline README.md]: Missing periods at the end.

**Comment by come-maiz** [comment]
:+1: 
Some suggestions on the writing, that's it.
I've enabled coveralls. We need to add some credentials to travis. I'll try to do it today or tomorrow.

**Comment by fgimenez** [comment]
Thanks! Fixed

> I've enabled coveralls. We need to add some credentials to travis. I'll try to do it today or tomorrow.

Never mind, on it now

**Pull request merged**